index.html

```diff
@@ -1,19 +1,129 @@
-<!doctype html>
+<!doctype html> 
 
 <html lang="en">
 <head>
   <meta charset="utf-8">
 
-  <title>My Resume</title>
+  <title>Katie Fitzpatrick Resume</title>
 
   <link rel="stylesheet" href="css/index.css">
 
+  <link href="https://fonts.googleapis.com/css?family=Lato:400,900|Yantramanav:300,400,700" rel="stylesheet">
   <!--[if lt IE 9]>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
   <![endif]-->
 </head>
 
 <body>
-  <h1>My Resume</h1>
+<div id=container>
```

Comment:
I think using unquoted html attributes like this may be somewhat unusual, so be aware if you're working on team projects that it may not be the team style

Reply:
got it, this was an accident/mistype - not done on purpose. Thanks for catching it